clients/java/src/main/java/io/zeebe/client/impl/worker/JobWorkerBuilderImpl.java

```diff
@@ -143,6 +152,7 @@ public JobWorker open() {
     ensureGreaterThan("timeout", timeout, 0L);
     ensureNotNullNorEmpty("workerName", workerName);
     ensureGreaterThan("maxJobsActive", maxJobsActive, 0);
+    ensureGreaterThan("minJobsActiveRatio", minJobsActiveRatio, 0.0f);
```

Comment:
Is it really necessary for this value to be greater than 0? What if I want the worker to only activate new jobs when it is done working on all other jobs? In the current implementation I could just set `minJobsActiveRatio` to something close to 0 like `0.00001` or so to make `minJobsActive` in the worker be rounded to 0, but that doesn't feel right. I think 0 could be an acceptable ratio. Another options I see is that we could allow directly setting the `minJobsActive`. WDYT?

Reply:
Agree with @korthout here. We may have use-cases where all the jobs fetched must be completed before polling again. If this validation is required, then we can probably have something like `ensureGreaterThanOrEqualTo` for this parameter.

Reply:
Thx for the feedback. I changed the code to allow 0 as suggested.